src/Tests/QueryDsl/Compound/FullText/MatchPhrasePrefix/MatchPhrasePrefixUsageTests.cs

```diff
@@ -3,33 +3,24 @@
 using Tests.Framework.MockData;
 using static Nest.Infer;
 
-namespace Tests.QueryDsl.FullText.Match
+namespace Tests.QueryDsl.FullText.MatchPhrasePrefix
 {
 	public class MatchPhrasePrefixUsageTests : QueryDslUsageTestsBase
 	{
 		public MatchPhrasePrefixUsageTests(ReadOnlyCluster i, EndpointUsage usage) : base(i, usage) { }
 
 		protected override object QueryJson => new
 		{
-			match = new
+			match_phrase_prefix = new
 			{
 				description = new
 				{
 					_name = "named_query",
 					boost = 1.1,
 					query = "hello worl",
 					analyzer = "standard",
-					fuzzy_rewrite = "constant_score_boolean",
-					fuzziness = "AUTO",
-					fuzzy_transpositions = true,
-					cutoff_frequency = 0.001,
 					prefix_length = 2,
```

Comment:
https://github.com/elastic/elasticsearch/blob/6d987a9b699c40376667fbd47ac8dda767ba1289/core/src/main/java/org/elasticsearch/index/query/MatchPhrasePrefixQueryBuilder.java#L203

Is this still a valid option?